app/controllers/users/verify_account_controller.rb

```diff
@@ -8,9 +8,9 @@ def index
 
     def create
       @verify_account_form = build_verify_account_form
+
       if @verify_account_form.submit
-        flash[:success] = t('account.index.verification.success')
-        redirect_to account_path
+        redirect_to sign_up_completed_url
```

Comment:
Would the no sp session scenario be better off in a separate PR? To generate the redirect url and service provider name we'll need additional behavior, I guess using the identity model? That is, assuming that we always want to show the user the completion page.